### PR TITLE
GFORMS-1132 - ATL Summary page button label logic

### PR DESCRIPTION
**Pull request by SandipKhedekar**

### CODE REVIEW

File: app/uk/gov/hmrc/gform/views/form/addToList.scala.html [hunk 1]
```diff
@@ -58,9 +59,13 @@
...
-@buttonLabel = @{
-    messages("button.saveAndContinue")
-}
+@button  = @{formTemplate.authConfig match {
+                    case Anonymous =>
+                        messages("button.continue")
+                    case _ =>
+                        messages("button.saveAndContinue")
+                    }
+            }
```
Comment (josef-vlach) [inline app/uk/gov/hmrc/gform/views/form/addToList.scala.html:68]: There is `continueLabelKey` method for this on `MaterialisedRetrievals`

**Review by josef-vlach — CHANGES_REQUESTED** [review]

File: app/uk/gov/hmrc/gform/views/form/addToList.scala.html [hunk 2]
```diff
@@ -124,7 +129,7 @@
       @{hiddenFieldsSnippets}
 
         <div class="govuk-button-group">
-           @govukButton(Button(content = Text(buttonLabel), inputType=Some("submit"), value=Some("SaveAndContinue"), name=Some("save")))
+           @govukButton(Button(content = Text(button), inputType=Some("submit"), value=Some(button), name=Some("save")))
```
Comment (girishkamat) [inline app/uk/gov/hmrc/gform/views/form/addToList.scala.html:132]: I dont think the button value is used in the backend. If not, can we remove it please?

File: app/uk/gov/hmrc/gform/views/form/addToList.scala.html (hunk 1 above)
Comment (SandipKhedekar) [reply app/uk/gov/hmrc/gform/views/form/addToList.scala.html]: Changed.

File: app/uk/gov/hmrc/gform/views/form/addToList.scala.html (hunk 2 above)
Comment (SandipKhedekar) [reply app/uk/gov/hmrc/gform/views/form/addToList.scala.html]: Agreed. Removed.

**Review by girishkamat — APPROVED** [review]

**Review by josef-vlach — APPROVED** [review]

**Pull request merged**